README.md

```diff
@@ -33,9 +33,10 @@ It's currently written in Objective-C / Swift and uses [VLCKit](https://code.vid
 
 ### VLC-iOS
 
-1. Run `pod install`.
-2. Open `VLC.xcworkspace`.
-3. Hit "Build and Run".
+1. Run `bundle install`.
+2. Run `bundle exec pod install`.
```

Comment:
yeah I forgot that this is better than just pod install can you explain again why though?

Reply:
Sure...

The project has some Ruby dependencies: fastlane, cocoapods, rubocop, xcode-install and xcodeproj. 

[Bundler](https://bundler.io) guarantees a consistent environment so that everybody (including CI) use the same versions of those dependencies. 

• `bundle install` installs all the dependencies on their expected versions. As defined by [Gemfile](https://code.videolan.org/videolan/vlc-ios/blob/master/Gemfile) and [Gemfile.lock](https://code.videolan.org/videolan/vlc-ios/blob/master/Gemfile.lock). (both already defined in the project)
• `bundle exec [command]` runs the command in a consistent environment with those dependencies installed.